### DISCUSSION TITLE
.ts files crashes SubtitleEdit

### DISCUSSION THREAD

**Discussion by doglover3920** [SubtitleEdit/subtitleedit#5944] [Q&A]
I am using SubtitleEdit 3.6.5
SubtitleEdit crashes when I try to use a .ts file. All files I tried.
It worked in the past.
Use mpv as player, but it crashes also when using VLC or directShow.

I found a workaround, by using MKVtoolnix and convert the .ts file to a MKV.

I suspect the problem is not within Subtitle Edit, but with utils on my Windows PC (Windows 8.1).  
The problem being that I have no idea what to update or re-install.

Does anybody have any ideas?

**Comment by niksedk** [comment]
Could you share the ts file?
I guess it only a specific ts file?

**Comment by doglover3920** [comment]
Not a specific .ts file.  All my .ts files.
recordings from a Linux/enigma receiver.
I made a 2 minutes cut and the file is here
http://epgspot.com/rytec_epg/test.ts

**Comment by niksedk** [comment]
Thx for the sample file - works well here however.... 
Could you try to re-download libmpv + ffmpeg (in Options - Settings - Videoplayer/waveform). Any better?
Could perhaps also be a Windows issue or gfx card driver issue.

**Reply by doglover3920** [reply]
Thanks.  This is what I feared then.
Something on my PC.

Redownloaded libmpv before and this had no effect.  Still crashes.

Will have to live with this, until I find something better.
